packages/widgets/src/components/ExchangeAccount/ExchangeAccount.tsx

```diff
@@ -0,0 +1,80 @@
+import { useTheme } from '@emotion/react';
+import {
+  Button,
+  CardBase,
+  CardContent,
+  CardHeader,
+  Icon,
+  Typography,
+} from '@ssa-ui-kit/core';
+import { BalancePieChart, WithLink } from '@components/AccountBalance';
+import { ExchangeAccountProps } from './types';
+import {
+  CardWrapper,
+  HeadWrapper,
+  PlatformWrapper,
+  RemoveButton,
+  StatusWrapper,
+  ChartWrapper,
+} from './styles';
```

Comment:
I think, we can use the next approach:
- rename each style to the component name, where it uses (CardBase, CardHeader, etc...)
- use `import * as S from './styles'`
- use `css={S.CardBase}` in the codebase

What do you think about this approach?

Reply:
Yes, that's a much better, good approach. Done